### DISCUSSION TITLE
MQTT client to control FullPageOS remotely

### DISCUSSION THREAD

**Discussion by olialb** [guysoft/FullPageOS#617] [Show and tell]
Hi guys,

I wrote a mqtt client in python to control my FullPageOS installation remote over my homeautomation.
You can 

- set the brigtness
- switch the backlight on/off 
- change the page which is currently shown to a set of preconfugured panles
- open a specific web page over mqtt
- trigger a shutdown or reboot

With pyautogui feature you can also send remote mouse clicks and keyboard commands over mqtt

May be this project is also usefull for someone else...

[mqttDisplayClient](https://github.com/olialb/mqttDisplayClient)

Have a good weekend
Oliver

**Comment by Noschvie** [comment]
Great idea! Do you plan to PR this feature?

**Reply by olialb** [reply]
Thank you! 
My first target is that its easy to install on FullPageOS over ssh. Try it!
Feel free to try and give feedback.
